src/Hedgehog.Experimental/Hedgehog.Experimental.fsproj

```diff
@@ -31,6 +33,7 @@
 
   <ItemGroup>
     <PackageReference Include="Hedgehog" Version="[0.13.0, 99]" />
+    <PackageReference Include="System.Collections.Immutable" Version="9.0.0" />
```

Comment:
Why is this needed?

Reply:
It is a standard .NET library.
Without it I cannot have `ImmutableDictionary`. And I cannot have F#-specific `Map` because `Map` cannot have `Type` as a key.